src/python/example.py

```diff
@@ -60,12 +60,12 @@ def __call__(self):
 simu.add_robot(robot)
 simu.add_checkerboard_floor(10., 0.1, 1., np.zeros((6,1)), "floor")
 
-# Add a camera to the end-effector of the manipulator
-camera = rd.gui.CameraOSR(simu, graphics.magnum_app(), 256, 256)
+# Add a camera sensor to the end-effector of the manipulator
+camera = rd.sensor.Camera(simu, graphics.magnum_app(), 256, 256)
 rot = dartpy.math.AngleAxis(3.14, [1., 0., 0.]).to_rotation_matrix()
 rot = rot.dot(dartpy.math.AngleAxis(1.57, [0., 0., 1.]).to_rotation_matrix())
-camera.attach_to("arm_link_5", dartpy.math.Isometry3(rotation=rot, translation=[0., 0., 0.1]))
-simu.add_camera(camera)
+camera.attach_to_body(robot.body_node("arm_link_5"), dartpy.math.Isometry3(rotation=rot, translation=[0., 0., 0.1]))
+simu.add_sensor(camera)
```

Comment:
What happens if we add a sensor without attaching it to a body first?

Reply:
It works as expected.. You can even attach it to a different body node while the simulation is running..